src/common.cpp

```diff
@@ -162,7 +163,7 @@ show_stackframe(const wchar_t msg_level, int frame_count, int skip_levels) {
 #else   // HAVE_BACKTRACE_SYMBOLS
 
 void __attribute__((noinline))
-show_stackframe(const wchar_t msg_level, int, int) {
+show_stackframe(const wchar_t msg_level, int frame_count, int skip_levels) {
     debug_shared(msg_level, L"Sorry, but your system does not support backtraces");
```

Comment:
I removed the named parameters recently because on platforms where backtrace symbols are not available those two symbols (`frame_count` and `skip_levels`) are not used (and generate a compiler warning to that effect). 

Is there a reason you are adding them back? Is this just outdated code being pulled back into `master` as a side effect?

Reply:
See https://github.com/fish-shell/fish-shell/commit/039c3c1673ca2f6974b5053675ac182bb80f6622

Reply:
That line was inadvertently reintroduced because I started this change on a different branch that didn't include your change. Plus it's unrelated to the purpose of this change so I'll remove it. Having said that your change isn't the preferred way to handle that situation. The preferred way is to use add `UNUSED(var_name);` lines to the body of the function to make it explicit that the arguments are knowingly being ignored.